.github/workflows/pull-request.yml

```
@@ -105,108 +105,28 @@ jobs:
-            exit 0
-          fi
-          
-          LABELS="$(gh api repos/${{ github.repository }}/issues/${{ github.event.pull_request.number }}/labels --jq '.[].name' | tr '\n' ' ')"
-          
-          if echo "$LABELS" | grep -qw "ignore-sensitive-files-pr"; then
-            echo "::notice::Skipping sensitive files check due to 'ignore-sensitive-files-pr' label."
-            echo "skip=true" >> $GITHUB_OUTPUT
-          else
-            echo "skip=false" >> $GITHUB_OUTPUT
-          fi
+          fetch-depth: 0 # Fetch all history for all branches and tags
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)